src/VisualStudio/IntegrationTest/TestUtilities/InProcess/TextViewWindow_InProc.cs

```diff
@@ -274,13 +281,18 @@ public bool IsLightBulbSessionExpanded()
        });
 
         public string[] GetLightBulbActions()
-            => ExecuteOnActiveView(view =>
+        {
+            return ThreadHelper.JoinableTaskFactory.Run(async () =>
             {
+                await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
+
+                var view = GetActiveTextView();
                 var broker = GetComponentModel().GetService<ILightBulbBroker>();
-                return GetLightBulbActions(broker, view).Select(a => a.DisplayText).ToArray();
+                return (await GetLightBulbActionsAsync(broker, view)).Select(a => a.DisplayText).ToArray();
```

Comment:
Missing a few explicit ConfigureAwaits in this file.

Reply:
Added